packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -85,6 +85,32 @@ export class TableColumnNumberText extends TableColumnTextBase {
         }
     };
 
+    public static convertDigitOptionsForDecimalUnitFormat(
+        decimalDigits?: number,
+        maximumDecimalDigits?: number
+    ): {
+            minimumFractionDigits?: number,
```

Comment:
Was there a reason to leave `minimumFractionDigits` as possibly `undefined`, or did that previous review comment get lost when this function moved?

Reply:
Whoops. Yes, got lost in the churn.